Error: A `label` alone cannot be used to provide an accessible label for a group of fields. Labels can only be associated with a single field using the `for` attribute. Use `fieldset` and `legend`.
Error: Use `aria-describedby` to reference a group description. Note: If the checkbox control is not nested in its `label`, wrapping the two elements within another element (such as a `div`) is preferable. Having a shared parent element gives designers more control over the eventual layout of the checkbox group than a `br` does.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/3/316d3a6c-67d4-42e8-aba7-c566b510239a.html b/output/Checkbox Group with Accessible Description/3/316d3a6c-67d4-42e8-aba7-c566b510239a.html
index 35255c5..fb3b784 100644
--- a/output/Checkbox Group with Accessible Description/3/316d3a6c-67d4-42e8-aba7-c566b510239a.html	
+++ b/output/Checkbox Group with Accessible Description/3/316d3a6c-67d4-42e8-aba7-c566b510239a.html	
@@ -1,8 +1,16 @@
-<label for="fruits">What fruits do you like?</label><br>
-<small>You can change this later</small><br>
-<input type="checkbox" id="bananas" name="fruit" value="bananas">
-<label for="bananas">Bananas</label><br>
-<input type="checkbox" id="oranges" name="fruit" value="oranges">
-<label for="oranges">Oranges</label><br>
-<input type="checkbox" id="apples" name="fruit" value="apples">
-<label for="apples">Apples</label>
\ No newline at end of file
+<fieldset aria-describedby="fruit-description">
+  <legend>What fruits do you like?</legend>
+  <small id="fruit-description">You can change this later</small>
+  <div>
+    <input type="checkbox" id="bananas" name="fruit[]" value="bananas">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruit[]" value="oranges">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruit[]" value="apples">
+    <label for="apples">Apples</label>
+  </div>
+</fieldset>
\ No newline at end of file
